samples/bluetooth/llpm/README.rst

```diff
@@ -0,0 +1,313 @@
+   Transmission Latency: 1052 (us)
+   Transmission Latency: 976 (us)
+   Transmission Latency: 1358 (us)
+   QoS conn event reports: channel index 0x10, CRC errors 0x00
+   Transmission Latency: 1281 (us)
+
+- For the slave::
+
+   ***** Booting Zephyr OS build v1.14.99-ncs3-snapshot2-2647-gd6e67554cfeb *****
+   Bluetooth initialized
+   LLPM mode enabled
+   Advertising successfully started
+   Scanning successfully started
+   Connection event reports enabled
+   Filter not match. Address: 1d:18:b1:84:fd:05 (random) connectable: 0
+   Filter not match. Address: 00:92:3f:a6:3f:48 (random) connectable: 0
+   Filter not match. Address: 02:ec:f9:bb:ec:27 (random) connectable: 0
+   Filter not match. Address: 3e:21:91:91:52:82 (random) connectable: 0
+   Filter not match. Address: 08:c6:a4:e0:72:e9 (random) connectable: 0
+   Filter not match. Address: 13:04:eb:f1:0b:46 (random) connectable: 0
+   Filter not match. Address: cb:01:1a:2d:6e:ae (random) connectable: 1
+   Filter not match. Address: 5c:f2:70:c2:3f:9f (random) connectable: 1
+   Connected as slave
+   Conn. interval is 80 units (1.25 ms/unit)
+   QoS conn event reports: channel index 0x1f, CRC errors 0x00
+   Service discovery completed
+   Press any key to start measuring transmission latency
+   QoS conn event reports: channel index 0x07, CRC errors 0x00
+   Transmission Latency: 80917 (us)
+   Transmission Latency: 80841 (us)
+   Transmission Latency: 80749 (us)
+   Transmission Latency: 80673 (us)
+   Transmission Latency: 80596 (us)
+   Transmission Latency: 80505 (us)
+   Transmission Latency: 80429 (us)
+   Transmission Latency: 80337 (us)
+   Transmission Latency: 80261 (us)
+   Transmission Latency: 80184 (us)
+   Transmission Latency: 80093 (us)
+   Transmission Latency: 80017 (us)
+   Transmission Latency: 79940 (us)
+   Transmission Latency: 79849 (us)
+   Connection interval updated: LLPM (1 ms)
+   Transmission Latency: 81604 (us)
+   Transmission Latency: 30181 (us)
+   Transmission Latency: 1098 (us)
+   QoS conn event reports: channel index 0x22, CRC errors 0x00
+   Transmission Latency: 1129 (us)
+   Transmission Latency: 1037 (us)
+   Transmission Latency: 930 (us)
+   Transmission Latency: 1312 (us)
+   Transmission Latency: 1083 (us)
+   Transmission Latency: 1007 (us)
+
+
+Dependencies
+*************
+
+This sample uses the following |NCS| libraries:
+
+* :ref:`latency_readme`
+
+In addition, it uses the following Zephyr libraries:
+
+* :file:`include/console.h`
+* :ref:`zephyr:kernel`:
+
+  * :file:`include/kernel.h`
+
+* :file:`include/misc/printk.h`
+* :file:`include/zephyr/types.h`
+* :ref:`zephyr:bluetooth_api`:
+
+  * :file:`include/bluetooth/bluetooth.h`
+  * :file:`include/bluetooth/conn.h`
+  * :file:`include/bluetooth/gatt.h`
+  * :file:`include/bluetooth/hci.h`
+  * :file:`include/bluetooth/uuid.h`
+  * :file:`include/bluetooth/scan.h`
+  * :file:`include/bluetooth/gatt_dm.h`
+
+Moreover, it uses the following Nrfx libraries:
```

Comment:
@greg-fer  , how do we document this properly? "It uses the BLE controller library in nrfxlib"?

Reply:
```suggestion
Moreover, it uses the following nrfxlib library::
```
I'd say it like this. The repetition doesn't hurt, and we keep consistency with the previous sentence about Zephyr libs.

Reply:
Should it rather be moved right below "This sample uses the following |NCS| libraries:"

Reply:
Need to check.

Reply:
OK, checked. Please keep the nrfxlib line, but move it to line 277 (move the Zephyr sentence down). The nrfxlib sentence should read:
```
This sample uses the following `nrfxlib`_ libraries:

* :ref:`nrfxlib:ble_controller`
```
We don't need the file mentioned under this ref.